drivers/volume/portworx/schedops/k8s-schedops.go

```diff
@@ -179,7 +179,9 @@ func (k *k8sSchedOps) ValidateVolumeSetup(vol *volume.Volume) error {
 		if ready := k8s.Instance().IsPodReady(p); !ready {
 			continue
 		}
-
+		//TODO: Debug why this fails intermittently
+		//Display pod status at this point.
+		logrus.Infof("Pod status for %s: %v", p.Name, p.Status)
```

Comment:
Change to

`logrus.Infof("Pod  [%s] %s ready for volume setup check. Status: %v",p.Namespace, p.Name, p.Status)`